lib/reference/shorthandData.js

```diff
@@ -1,10 +1,9 @@
 'use strict';
 
-/** @type {Record<string, string[]>} */
```

Comment:
[note] I think this `@type` is needless because the auto-completion via property names works.